pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1655,11 +1655,26 @@ cdef class BusinessHour(BusinessMixin):
     >>> ts + pd.offsets.BusinessHour(end=dt_time(19, 0))
     Timestamp('2022-08-08 10:00:00')
 
-    The parameter ``normalize`` equal True forces shift to midnight.
+    Passing the parameter ``normalize`` equal True, you shift the start
+    of next business hour to midnight.
 
     >>> ts = pd.Timestamp(2022, 12, 9, 8)
     >>> ts + pd.offsets.BusinessHour(normalize=True)
     Timestamp('2022-12-09 00:00:00')
+
+    You can divide your business day hours into several parts.
+
+    >>> import datetime as dt
+    >>> freq = pd.offsets.CustomBusinessHour(start=["06:00", "10:00", "15:00"],
```

Comment:
I guess this should still be `BusinessHour`?

Reply:
Oops! I corrected my mistake. The copy-and-paste method can be dangerous sometimes.